README.md

```diff
@@ -50,12 +50,12 @@ dlp_incident = c.dlp_incidents.get(44010)
 c = client.Client(api_key='your-api-key-here', instance_name='yourtenantname')
 next_page_token = None
 incidents = []
-threats, next_page_token, total_incident_count = c.threats.list(page_token=next_page_token)
-incidents.append(threats)
-while next_page_token != None:
+while True:
     threats, next_page_token, total_incident_count = client.threats.list(page_token=next_page_token)
-    incidents.append(threats)
-
+    if threats:
```

Comment:
Added this check to avoid adding to incidents if there are no threats available. Not sure if this might be encountered  though.

Reply:
This might not be needed as the call returns an empty array if there's nothing available - you will have to use `extend` instead of `append` though I think.

Reply:
Right, will change it.